### ISSUE TITLE
"Avoid unnecessary zero-length array allocations" code fix cannot VB negative initialization

### ISSUE THREAD

**Issue by davkean** [dotnet/roslyn-analyzers#1863]
#### Diagnostic ID

CA1825

#### Repro steps

``` VB
	Dim TargetNamespaces(-1) As String
```

#### Expected behavior

``` VB
	Dim TargetNamespaces As String() = Array.Empty(Of String)
```

#### Actual behavior

Nothing, it does not change the above code
